src/index.html

```diff
@@ -1,12 +1,383 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="moyo">
```

Comment:
the same

Reply:
Hey. I need class for body, because without it, i can't position my catalog.
As i know, addштп margin to block is forbidden.
I created class for body and class "moyo__catalog" for my catalog.

Reply:
So you can position your `.catalog `selector by using `flex`. In this way you don't need additional classes `.moyo` and `.moyo__catalog`